rules/aip0192/leading_comments_test.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0192
+
+import (
+	"testing"
+
+	"github.com/googleapis/api-linter/rules/internal/testutils"
+)
+
+func TestOnlyLeadingComments(t *testing.T) {
+	tests := []struct {
+		testName string
+		Detached string
+		Trailing string
+		problems testutils.Problems
+	}{
+		{"ValidNone", "", "", testutils.Problems{}},
+		{"ValidInternalDetached", "// (-- detached --)", "", testutils.Problems{}},
+		{"ValidInternalTrailing", "// (-- trailing --)", "", testutils.Problems{}},
```

Comment:
This is still "detached" comments, not the trailing one.

Reply:
Done.